acceptancetests/jujupy/client.py

```diff
@@ -1877,7 +1877,7 @@ def action_do(self, unit, action, *args):
 
         output = self.get_juju_output("run-action", *args)
         action_id_pattern = re.compile(
-            'Action queued with id: ([a-f0-9\-]{36})')
+            'Action queued with id: ([a-z0-9\-]+)')
```

Comment:
Worth doing an OR to account for old uuids and new <applicationname>-N ids?
The regexp for <appname>-N would be

`((?:[a-z][a-z0-9]*(?:-[a-z0-9]*[a-z][a-z0-9]*)*)-)(?:0|[1-9][0-9]*)`